marda_registry/data/filetypes/biologic-mpr.yml

```
@@ -1,10 +1,38 @@
 ---
-id: biologic-mpr
-name: BioLogic MPR file
-associated_vendors:
-    - BioLogic
+id: >-
+    biologic-mpr
+name: >-
+    BioLogic MPR file
+description: >-
+    A binary data file created by BioLogic cyclers, containing time- and frequency-domain
+    data pertaining to many electrochemical techniques, such as voltammetry,
+    potentiometry, coulometry and impedance spectroscopy.
```

```suggestion
    potentiometry, coulometry and impedance spectroscopy.
associated_vendors:
    - BioLogic
```

Sorry, think I mangled this one myself on the merge